Statsman/PeriodStatisticsService.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Statsman.Core.TimeSplitting;
+using Statsman.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Watchman.Common.Models;
+using Watchman.Cqrs;
+using Watchman.DomainModel.Messages;
+using Watchman.DomainModel.Messages.Queries;
+using Watchman.Integrations.Quickchart;
+
+namespace Statsman
+{
+    public class PeriodStatisticsService
+    {
+        private readonly IQueryBus queryBus;
+        private readonly TimeSplittingService timeSplittingService = new TimeSplittingService(); //TODO IoC
+        private readonly ChartsService _chartsService = new ChartsService();
+
+        public PeriodStatisticsService(IQueryBus queryBus)
+        {
+            this.queryBus = queryBus;
+        }
```

Comment:
używając TODO IoC rozumiem, że chodzi, że timeSplittingService powinno być pobierane z IoC? Jeśli tak, to czemu po prostu tak nie uczynisz? Z tego co ja widzę obiekt klasy PeriodStatisticsService jest pobierany z context (tak mi się wydaje bo widzę w ctorze IQueryBus), więc do niego może być również przekazany timeSplittingService.